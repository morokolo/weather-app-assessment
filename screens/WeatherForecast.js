import React from "react";
import { StyleSheet, Text, View } from "react-native";

export default function WeatherForecast() {
  return (
    <View>
      <Text>Hello</Text>
    </View>
  );
}

const styles = StyleSheet.create({});
