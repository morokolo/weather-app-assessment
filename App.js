import { StatusBar } from "expo-status-bar";
import React from "react";
import { StyleSheet, Text, View } from "react-native";
import WeatherForecast from "./screens/WeatherForecast";

export default function App() {
  return <WeatherForecast />;
}

const styles = StyleSheet.create({});
