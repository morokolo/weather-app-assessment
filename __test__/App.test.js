import "react-native";
import React from "react";
import App from "../App";
import renderer from "react-test-renderer";
import WeatherForecast from "../screens/WeatherForecast";

it("should render the App component", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
