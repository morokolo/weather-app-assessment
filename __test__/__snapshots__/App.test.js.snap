// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the App component 1`] = `
<View>
  <Text>
    Hello
  </Text>
</View>
`;
